package.json

```diff
@@ -26,7 +26,8 @@
     "build": "npm run clean && microbundle --entry src/index.js --output dist --name goober --no-sourcemap && npm run size-check",
     "debug": "microbundle --entry src/index.js --output debug --name goober --no-sourcemap --no-compress",
     "dev": "npm run clean && microbundle watch --entry src/index.js --output dist --name goober",
-    "deploy": "npm run build && npm publish"
+    "deploy": "npm run build && npm publish",
+    "format": "prettier \"**/*.{js,ts,tsx,md}\" --write"
```

Comment:
Do you feel like we would still need this script, since `lint-staged` would take care of the formatting? I'm fine with it, just making sure this won't introduce another level of "wait, should I use format? Or..?"

Reply:
depends.

I usually keep this script for following scenarios:
- someone might skip the hook, no checking on CI, merge -> handy for reformat
- bumping to new versions of prettier that might introduce some regressions, run `format` done

Contributors should not care and follow pre-commit pipeline without noticing.

Reply:
Alright, fair enough. 👍